etc/kayobe/ansible/install-pre-commit-hooks.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Install pre-commit hooks
+  hosts: localhost
+  gather_facts: false
+  vars:
+    pre_commit_version: 3.5.0
+  tasks:
+    - name: Install pre-commit hooks
+      block:
+        - name: Install pre-commit hooks into kayobe virtual env
+          ansible.builtin.pip:
+            name: pre-commit
+            version: "{{ pre_commit_version  }}"
+            virtualenv: "{{ lookup('ansible.builtin.env', 'VIRTUAL_ENV') }}"
```

Comment:
Is there an issue here that it forces us to use a virtualenv when developing SKC? e.g. I don't necessarily have one set up when writing docs.

Reply:
I have updated this to support installation without a virtualenv if one isn't present.